src/features/Students/StudentsTable/_test_/index.test.jsx

```diff
@@ -44,7 +56,9 @@ describe('Student Table', () => {
     ];
 
     const component = render(
-      <StudentsTable data={data} count={data.length} columns={getColumns()} />,
+      <Provider store={store}>
+        <StudentsTable data={data} count={data.length} columns={getColumns()} />
+      </Provider>,
     );
 
     // Check if the table rows are present
```

Comment:
Is this inline comment necessary? @AuraAlba

Reply:
Not. I removed them